flecs_ecs_sys/src/flecs.h

```diff
@@ -160,30 +160,30 @@
-#define FLECS_STATS         /**< Access runtime statistics */
-#define FLECS_MONITOR       /**< Track runtime statistics periodically */
-#define FLECS_METRICS       /**< Expose component data as statistics */
-#define FLECS_ALERTS        /**< Monitor conditions for errors */
-#define FLECS_SYSTEM        /**< System support */
-#define FLECS_PIPELINE      /**< Pipeline support */
-#define FLECS_TIMER         /**< Timer support */
-#define FLECS_META          /**< Reflection support */
-#define FLECS_META_C        /**< Utilities for populating reflection data */
-#define FLECS_UNITS         /**< Builtin standard units */
-#define FLECS_EXPR          /**< Parsing strings to/from component values */
-#define FLECS_JSON          /**< Parsing JSON to/from component values */
-#define FLECS_DOC           /**< Document entities & components */
-#define FLECS_COREDOC       /**< Documentation for core entities & components */
-#define FLECS_LOG           /**< When enabled ECS provides more detailed logs */
-#define FLECS_APP           /**< Application addon */
-#define FLECS_OS_API_IMPL   /**< Default implementation for OS API */
-#define FLECS_HTTP          /**< Tiny HTTP server for connecting to remote UI */
-#define FLECS_REST          /**< REST API for querying application data */
+//#define FLECS_MODULE        /**< Module support */
+//#define FLECS_PARSER        /**< String parser for queries */
+//#define FLECS_PLECS         /**< ECS data definition format */
+//#define FLECS_RULES         /**< Constraint solver for advanced queries */
+//#define FLECS_SNAPSHOT      /**< Snapshot & restore ECS data */
+//#define FLECS_STATS         /**< Access runtime statistics */
+//#define FLECS_MONITOR       /**< Track runtime statistics periodically */
+//#define FLECS_METRICS       /**< Expose component data as statistics */
+//#define FLECS_ALERTS        /**< Monitor conditions for errors */
+//#define FLECS_SYSTEM        /**< System support */
+//#define FLECS_PIPELINE      /**< Pipeline support */
+//#define FLECS_TIMER         /**< Timer support */
+//#define FLECS_META          /**< Reflection support */
+//#define FLECS_META_C        /**< Utilities for populating reflection data */
+//#define FLECS_UNITS         /**< Builtin standard units */
+//#define FLECS_EXPR          /**< Parsing strings to/from component values */
+//#define FLECS_JSON          /**< Parsing JSON to/from component values */
+//#define FLECS_DOC           /**< Document entities & components */
+//#define FLECS_COREDOC       /**< Documentation for core entities & components */
+//#define FLECS_LOG           /**< When enabled ECS provides more detailed logs */
+//#define FLECS_APP           /**< Application addon */
+//#define FLECS_OS_API_IMPL   /**< Default implementation for OS API */
+//#define FLECS_HTTP          /**< Tiny HTTP server for connecting to remote UI */
+//#define FLECS_REST          /**< REST API for querying application data */
```

Comment:
You could just define `FLECS_CUSTOM_BUILD` and then you wouldn't need to modify this file.

Reply:
true, would be easier when upgrading as well to not have to keep doing this